ruby/lib/item_type.rb

```diff
@@ -0,0 +1,35 @@
+def aged_brie(item)
```

Comment:
I might have named this `handle_aged_brie` (same for the rest), so it's more apparent from the method name what is going on.